### PR TITLE
Feature/#522 sf circle icon on sf product card

### PR DESCRIPTION
**Pull request by martaradziszewska**
# Related issue
https://github.com/DivanteLtd/storefront-ui/issues/522

# Scope of work
Implement SfCircleIcon on SfProductCard
Added transition to button whether product is added to cart or not
Create new story of SfProductCard with SfCircleIcon

# Screenshots of visual changes

Peek 2019-12-13 08-14

# Checklist

- [x] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

**Comment by filrak** [comment]
Changes as we discussed ;)

File: packages/vue/public/assets/add_to_cart.svg [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-<path fill-rule="evenodd" clip-rule="evenodd" d="M21.6229 3.947C22.3418 3.947 23.0366 4.29388 23.4833 4.86461L23.4832 4.86468C23.9551 5.43541 24.1034 6.17957 23.9299 6.8753L22.0695 14.5157C21.8214 15.5825 20.8534 16.3267 19.7624 16.3267H9.86547C8.77343 16.3267 7.83161 15.5825 7.55835 14.5157L4.53232 2.45966C4.4335 2.08758 4.11083 1.83952 3.73874 1.83952H1.20882C0.786334 1.83952 0.439453 1.49165 0.439453 1.07015C0.439453 0.647662 0.787326 0.300781 1.20882 0.300781H3.73874C4.83078 0.300781 5.7726 1.04495 6.04586 2.11179L6.51677 3.947H11.9507V5.5101H6.91403L9.07191 14.1667C9.17074 14.5388 9.49341 14.7869 9.86549 14.7869H19.7634C20.1355 14.7869 20.4582 14.5388 20.557 14.1667L22.4174 6.52638L22.4176 6.52656C22.4912 6.2785 22.4176 6.03044 22.2693 5.83277C22.1201 5.63413 21.872 5.5101 21.624 5.5101H18.0734V3.947H21.6229Z" fill="white"/>
+<svg width="25" height="23" viewBox="0 0 25 23" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (przspa) [inline packages/vue/public/assets/add_to_cart.svg]: you should update it in shared/icons/icons.js :)

Comment (martaradziszewska) [reply packages/vue/public/assets/add_to_cart.svg]: yes, I updated this file too

**Comment by patzick** [comment]
🚀

**Pull request merged**